src/renderer/components/atoms/text/types.ts

```diff
@@ -0,0 +1,53 @@
+import type { ReactNode } from 'react';
+
+export type TextTag = 'h1' | 'h2' | 'h3' | 'h4' | 'span' | 'p';
+export type TextUsage =
+  | 'heading-1-interface'
+  | 'heading-2-interface'
+  | 'heading-3-interface'
+  | 'heading-4-interface'
+  | 'body-interface'
+  | 'body-small-interface'
+  | 'heading-1-document'
+  | 'heading-2-document'
+  | 'heading-3-document'
+  | 'heading-4-document'
+  | 'body-document'
+  | 'body-small-document'
+  | 'error-document'
+  | 'quote-document';
+export type TextVariant = 'bold' | 'normal' | 'italic' | '';
+export type TextColor =
+  | 'primary'
+  | 'secondary'
+  | 'blueWhite'
+  | 'redWhite'
```

Comment:
if the `as` prop is using kebab-case, let's use it here as well
```suggestion
  | 'blue-white'
  | 'red-white'
```

Reply:
right, done!